zscaler_private_access/README.md

```diff
@@ -61,9 +61,9 @@ The Zscaler Private Access check is included in the [Datadog Agent][1] package.
 3. Click **Add**.
 4. In the **Log Receiver** tab, configure the following:
     - **Name**: Provide a name for the log receiver.
-    - **Domain or IP Address**: Enter the public IP or hostname of the Datadog Agent that will receive the logs.
-    - **TCP Port**: Specify an open port on the Datadog Agent for receiving ZPA logs.
-    - **TLS Encryption**: Keep it disabled.
+    - **Domain or IP Address**: If TLS encryption is enabled, provide the `public IP` or `hostname` of the `syslog-ng` server; otherwise, provide the `public IP` or `hostname` of the Datadog agent.
+    - **TCP Port**: If TLS encryption is enabled, specify an open port on the `syslog-ng` server; otherwise, specify an open port on the Datadog agent.
```

Comment:
```suggestion
    - **TCP Port**: If TLS encryption is enabled, specify an open port on the `syslog-ng` server; otherwise, specify an open port on the Datadog Agent.
```